gamemodes/terrortown/gamemode/client/cl_status.lua

```diff
@@ -8,14 +8,19 @@ STATUS.active = {}
 
 ---
 -- Registers a @{STATUS}
--- @param number id The index of the new status
+-- @param number id the index of the new status
 -- @param table data
 -- @return boolean whether the creation was successfully
 -- @realm client
 function STATUS:RegisterStatus(id, data)
 	if STATUS.registered[id] ~= nil then  -- name is not unique
 		return false
 	end
+	
+	-- support single and multible icons per status effect
+	if data.hud.GetTexture then
+		data.hud = {data.hud}
```

Comment:
lol, why not using `istable(data.hud)` check?

Reply:
Since materials are tables too, I have to check if it is a material. The table check returns true for both.

Reply:
lol ok, nm